### DISCUSSION TITLE
想起宋芳蓉 | /cn/2005/05/song-fangrong/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1568] [Comments]
今天“五四”，本想写点什么纪念一下，突然间想起一个人——宋芳蓉。
[&hellip;] 2002年暑假，我去参加了宜昌市西陵剧场的一场晚会，是关于宜昌市教育局资助50名品学兼优的贫困大学新生的，有幸代表50位同学讲了几句话；今天之所以想起宋芳蓉，是因为当时也很荣幸见到了她，时间也真快，都过了三年了，如今看见她获得了“五四青年奖章”，心里为她高兴，但更多的是崇敬。
[&hellip;] 如果不是在网 &hellip;

https://yihui.org/cn/2005/05/song-fangrong/

**Comment by think-crow** [comment]


**Reply by yihui** [reply]
已修正 404 链接，多谢报告。

**Reply by think-crow** [reply]
哈哈！下次你再客气就不给你说了！

你在我这熟得很！

**Reply by yihui** [reply]
下次要是能直接在 Github 上帮我修正就更好了。失效的陈年老链接通常都可以通过 https://web.archive.org 找到。示例：https://github.com/yihui/yihui.org/commit/7cc9b0a277c6869f1da3bc360496a879a4c414c6
